guides/common/modules/proc_upgrading-smartproxy-server.adoc

```diff
@@ -137,5 +137,44 @@ include::snip_steps-needs-reboot.adoc[]
 
 . Optional: If you made manual edits to DNS or DHCP configuration files, check and restore any changes required to the DNS and DHCP configuration files using the backups made earlier.
 ifdef::orcharhino,satellite[]
-. Optional: If you use custom repositories, ensure that you enable these custom repositories after the upgrade completes.
+. Optional: If you use custom repositories,ensure that you enable these custom repositories after the upgrade completes.
 endif::[]
+
+.Upgrading {SmartProxyServersTitle} Using Remote Execution
+
+. Create a backup.
++
+* On a virtual machine, take a snapshot.
+* On a physical machine, create a backup.
++
+For more information on backups, see {AdministeringDocURL}backing-up-satellite-server-and-capsule-server[Backing Up {ProjectServer} and {SmartProxyServer}] in the _Administering {ProjectName} {ProjectVersionPrevious}_ guide.
```

Comment:
There are also filesystems which can take snapshots. I think admins have sufficient knowledge on what they can or can't do, or otherwise the link document does a much better job of describing it. So would it be sufficient to say:
```suggestion
. Create a backup or take a snapshot.
+
For more information on backups, see {AdministeringDocURL}backing-up-satellite-server-and-capsule-server[Backing Up {ProjectServer} and {SmartProxyServer}] in the _Administering {ProjectName} {ProjectVersionPrevious}_ guide.
```